packages/x-date-pickers/src/internals/components/CalendarOrClockPicker/CalendarOrClockPicker.tsx

```diff
@@ -148,6 +148,18 @@ export function CalendarOrClockPicker<TDate, View extends CalendarOrClockPickerV
     [isMobileKeyboardViewOpen, onViewChange, toggleMobileKeyboardView],
   );
 
+  if (process.env.NODE_ENV !== 'production') {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const warnedOnceNotValidOpenTo = React.useRef(false);
+    if (!warnedOnceNotValidOpenTo.current && !views.includes(openTo)) {
+      console.warn(
+        `MUI: \`openTo="${openTo}"\` is not a valid prop.`,
+        `The possible \`view\` values are \`["${views.join('", "')}"]\`.`,
```

Comment:
If the dev make the error, it might be because they did not get the link between `openTo` and `views`

```suggestion
        `It must be an element of \`views=["${views.join('", "')}"]\`.`,
```